### PR TITLE
Add ADME Distribution data from TDC

### PR DESCRIPTION
**Pull request by phalem**
I finish ADME Distribution data from TDC:
BBB (Blood-Brain Barrier), Martins et al.
PPBR (Plasma Protein Binding Rate), AstraZeneca
VDss (Volumn of Distribution at steady state), Lombardo et al.

### CODE REVIEW

**Review by phalem — COMMENTED** [review]
Add benchmark field

File: data/plasma_protein_binding_rate_astrazeneca/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+    - id: Species
+      type: Other
+      names:
+          - species
+      description: species in which the measurement was carried out
+    - id: chembl_id
```
Comment (kjappelbaum) [inline data/plasma_protein_binding_rate_astrazeneca/meta.yaml:30]: @MicPie do we want to keep this for the prompts? will learning the IDs help the model?

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
one question about ids, otherwise lgtm

File: data/plasma_protein_binding_rate_astrazeneca/meta.yaml (hunk 1 above)
Comment (MicPie) [reply data/plasma_protein_binding_rate_astrazeneca/meta.yaml:30]: maybe not for the prompt but for merging it into the KG?

Comment (kjappelbaum) [reply data/plasma_protein_binding_rate_astrazeneca/meta.yaml:30]: yes, so shall we add an attribute that prevents sampling? Probably I'd anyhow merge on SMILES?

Comment (kjappelbaum) [reply data/plasma_protein_binding_rate_astrazeneca/meta.yaml:30]: cause currently we have no mechanism to prevent sampling of such identifiers

Comment (MicPie) [reply data/plasma_protein_binding_rate_astrazeneca/meta.yaml:30]: yeah, very likely we will merge only in SMILES, and we could exclude cols with the id `chembl_id`.

Comment (MicPie) [reply data/plasma_protein_binding_rate_astrazeneca/meta.yaml:30]: We could easily prevent sampling by not using the column in the template, or am I missing something?

Comment (MicPie) [reply data/plasma_protein_binding_rate_astrazeneca/meta.yaml:30]: added flag for it

**Review by kjappelbaum — APPROVED** [review]

**Pull request merged**